app/controllers/meals_controller.ts

```diff
@@ -1,16 +1,83 @@
-import Meal from '#models/meal'
+import HashesMeal from '#models/hashes_meal'
+import WebsiteHash from '#models/website_hash'
 import type { HttpContext } from '@adonisjs/core/http'
+import logger from '@adonisjs/core/services/logger'
 
 export default class MealsController {
   /**
    * Display a list of resource
    */
   async index({ response }: HttpContext) {
     try {
-      const meals = await Meal.all()
+      const lastHash = await WebsiteHash.query().orderBy('createdAt', 'desc').first()
+      if (!lastHash) {
+        return response.status(200).json([])
+      }
+      let isMenuOnline = true
+      let todayMeals = await getMealsByHash(lastHash.hash)
+
+      if (todayMeals.length === 0) {
+        const secondLastHash = await WebsiteHash.query()
+          .orderBy('createdAt', 'desc')
+          .offset(1)
+          .first()
+        if (!secondLastHash) {
+          return response.status(200).json([])
+        }
+        isMenuOnline = false
+        todayMeals = await getMealsByHash(secondLastHash.hash)
+      }
+
+      const meals = todayMeals.map((singleMeal) => ({
+        ...singleMeal.meal.serialize(),
+        price: singleMeal.price,
+        size: singleMeal.size,
+      }))
+
+      return response.status(200).json({ meals, isMenuOnline: isMenuOnline })
```

Comment:
`{ meals, isMenuOnline: isMenuOnline }` is the same as `{ meals, isMenuOnline }`

Reply:
you can't run from the watchful eye of the mr. Shark